src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,250 @@
+package duke;
+import duke.addable.*;
+import duke.exception.*;
```

Comment:
Try to avoid the usage of "*" while importing classes.  Best is to use the name of the specific class that you want to import. This improves the code readability.